### PR TITLE
Add DevLab support

### PR DESCRIPTION
**Pull request by ghost**
Adds support for [DevLab](https://github.com/TechnologyAdvice/DevLab) for :whale: development using [Node 4 LTS (Argon)](https://nodejs.org/en/blog/release/v4.2.0/).

Here are instructions for getting set-up:
- Install VirtualBox and Homebrew
- Brew install Docker and Docker Machine
- With Docker Machine create a VM called `vbox-dev` using the Oracle VirtualBox driver
- Follow along with [this post](http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/) and run the demo project to be sure your environment is configured properly.
- With environment configured you can run `lab start` to begin development (see `devlab.yml` for available tasks)

Once started, WDS will bind port `8080` on the client to the host. `0.0.0.0` is used in place of `localhost` to actually make that work. Everything else should run the same.

While I was in there I also moved a number of dependencies into the `devDependencies` section of `package.json` for faster `--production` installs.

screen shot 2015-12-06 at 11 55 23 am

@levithomason @Fluidbyte

### CODE REVIEW

File: devlab.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# DEVLAB-VERSION 1.5.3
+
+#
+# Stardust DevLab YAML
+#
+# http://fluidbyte.net/docker/devlab/nodejs/2015/10/15/containerize-your-local-dev-in-minutes-with-devlab/
+# https://github.com/TechnologyAdvice/stardust/
+#
+
+from: node:argon
```
Comment (ksafranski) [inline devlab.yml:11]: We have a node:4 container you could use for consistency: `technologyadvice/node-build:4` [see hub-template](https://github.com/TechnologyAdvice/hub-template/blob/staging/devlab.yml)

Comment (TomFrost) [inline devlab.yml:11]: Not only for consistency-- our container is about a fifth of the size of the official one. They insist on building on top of a full Debian install.

Comment (ghost) [inline devlab.yml:11]: Good to know! Will update before merge.

**Comment by ghost** [comment]
Going to keep this open for a little while longer while I get things working on our skinny nix image and dogfood the approach. We're going to look at adding an NVM dotfile to help ensure environment consistency meanwhile as needed.

**Comment by ksafranski** [comment]
:+1:

**Comment by ghost** [comment]
See #135 for more. PR still a useful reference.

**Pull request closed without merging**